Cargo.toml

```diff
@@ -30,6 +30,8 @@ libc = { version = "0.2.62", default-features = false, optional = true }
 
 [features]
 parallel = ["dep:libc", "dep:jobserver", "dep:once_cell"]
+# enables OutputKind::Stderr which requires rustc 1.74.0 -- currently above msrv
```

Comment:
Hm, maybe it's time to bump the MSRV to 1.75 (RHEL 8.10 & 9.4, Ubuntu 24.04LTS seem to have settled on that version).

Reply:
This would certainly rock for me personally

Reply:
[Elsewhere](https://github.com/rust-lang/cc-rs/pull/1140#issuecomment-2213008655) I've recommended we switch to stable-5 (which I believe is in line with `libc`'s stated MSRV... even if it's actual MSRV is preposterously old) which would give us 1.74.

Reply:
Just to verify, the only reason this is behind a feature is the MSRV situation, right?

Reply:
correct